test/common/stats/thread_local_store_test.cc

```diff
@@ -189,6 +260,152 @@ TEST_F(StatsThreadLocalStoreTest, BasicScope) {
   // Includes overflow stat.
   EXPECT_CALL(*this, free(_)).Times(5);
 }
+TEST_F(StatsThreadLocalStoreTest, BasicSingleHistogramMerge) {
```

Comment:
Do we have any test where we actually look at the merged results to see if the percentiles are as expected?  I scanned through and didn't see one but I might have missed it.

Reply:
I have added it in one of the tests now.